app/src/main/res/values/strings.xml

```diff
@@ -810,4 +810,9 @@
     <!-- Open source licenses page title
     The first parameter is the app name -->
     <string name="open_source_licenses_title">%s | OSS Libraries</string>
+
+    <!-- Content description of the tab counter toolbar button when one tab is open -->
+    <string name="tab_counter_content_description_one_tab">1 tab</string>
+    <!-- Content description of the tab counter toolbar button when multiple tabs are open. First parameter will be replaced with the number of tabs (always more than one) -->
+    <string name="tab_counter_content_description_multi_tab">%d tabs</string>
```

Comment:
I fear that this is not going to work in all languages. Some use different words based on other quantities than 1 and >1.

CC @Delphine

Reply:
"Fun": https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals

Reply:
We already do this with other strings. See #4655.

@ekager Could you add these strings to the issue after merging?

Reply:
Yes I can add those strings there after merging, still need a review first :)